.clang-format-ignore

```diff
@@ -1,13 +1,3 @@
-# C++ code generated from Slice files
```

Comment:
I could not get this to work. The solution is to generate a clang-format off in the .h files.